webmails/roundcube/config/php.ini

```diff
@@ -2,3 +2,7 @@ expose_php=Off
 date.timezone={{ TZ }}
 upload_max_filesize = {{ MAX_FILESIZE }}M
 post_max_size = {{ MAX_FILESIZE }}M
+suhosin.session.encrypt=Off
```

Comment:
As discussed on #mailu-dev I don't think that's a good idea.

I suggest we remove suhosin and add snuffleupagus instead

Reply:
I will remove it and will add snuffleupagus. But this does mean we must test if it breaks anything.

Reply:
I have removed it, but snuffleupagus is not available for alpine 3.16. the edge repo only has a package for php7. We use php8.